server.go

```diff
@@ -353,6 +353,10 @@ func (svr *Server) Serve() error {
 	for {
 		pktType, pktBytes, err = svr.serverConn.recvPacket(svr.pktMgr.getNextOrderID())
 		if err != nil {
+			// Check whether the connection terminated cleanly in-between packets.
+			if err == io.EOF {
+				err = nil
```

Comment:
Hm… I usually find these “set an error to return later” patterns to be error prone, but I think reworking this whole `Serve` function just to avoid the pattern already in use doesn’t make sense.

I’ll chalk it up as something else for the inevitable rewrite in v2.

Reply:
Acknowledged.